tutorials/developers/dev_pipeline/dev_pipeline.tex

```diff
@@ -0,0 +1,62 @@
+\documentclass{tufte-handout}
+\usepackage{../braph2_dev}
+%\geometry{showframe} % display margins for debugging page layout
+
+\title{Implement a new Pipeline}
+
+\author[The BRAPH~2 Developers]{The BRAPH~2 Developers}
+
+\begin{document}
+
+\maketitle
+
+\begin{abstract}
+\noindent
+This is the developer tutorial for implementing a new pipeline. 
+In this tutorial, we will explain how to create a \fn{*.braph2} file to change the steps of a pipeline in braph2genesis. Here, we will use as an example the pipeline \code{Pipeline Structural Multiplex Comparison BUD} (the pipeline for comparing two groups of subjects using structural multiplex data and binary undirected graphs at fixed densities).
+\end{abstract}
+
+\tableofcontents
+
+%%%%% %%%%% %%%%% %%%%% %%%%%
+\clearpage
+\section{Implementation of the pipeline}
+\subsection{Pipeline Structural Multiplex Comparison BUD}
+
+
+
+\begin{lstlisting}[
+	label=cd:m:P_SMCBUD:all,
+	caption={
+		The pipeline for comparing two groups of subjects using structural multiplex data and binary undirected graphs at fixed densities.
+		}
+]
+%% Pipeline Structural Multiplex Comparison BUD ¥\circled{1}\circlednote{1}{specifies the name of the piepline.}¥
+
+% This is the pipeline script to compare two groups of subjects using structural multiplex data and binary undirected graphs at fixed densities. ¥\circled{2}\circlednote{2}{The descriptions for each step are annotated.}¥
+% 1. It loads a brain atlas from an XLS file (e.g., desikan_atlas.xlsx).
+% 2. It loads the data of two groups of subjects from two directories (e.g., ST_MP_group_1_XLS and ST_MP_group_2_XLS).
+% 3. It analyzes the first group using structural multiplex analyses (ST_MP) based on binary unidrected graphs at fixed densities (BUD).
+% 4. It analyzes the second group using the same parameters selected for the first group.
+% 5. It compares the results of the two analyses.
+
+%% Brain Atlas
+ba = ImporterBrainAtlasXLS('WAITBAR', true).get('GET_FILE').get('BA'); ¥\circled{3}\circlednote{3}{loads the brain atlas.}¥
+
+%% Groups
+gr1 = ImporterGroupSubjectST_MP_XLS('BA', ba, 'WAITBAR', true).get('GET_DIR').get('GR'); ¥\circled{4}\circlednote{4}{loads group 1 from XLS.}¥
+gr2 = ImporterGroupSubjectST_MP_XLS('BA', ba, 'WAITBAR', true).get('GET_DIR').get('GR'); ¥\circled{5}\circlednote{5}{loads group 2 from XLS.}¥
+
```

Comment:
Please, add a third group. Like in the tutorial of the previous year: http://braph.org/tutorials/for-developers/new-pipeline/